Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting.yml

```diff
@@ -0,0 +1,1496 @@
+      isArray: false
+      name: from
+      required: false
+      secret: false
+    - default: false
+      description: A timestamp (milliseconds) or relative time string (for example:'now' or 1661510185000). Filter for data that appears before this time. By default value is 'now'.
+      isArray: false
+      name: to
+      required: false
+      secret: false
+    - default: false
+      description: The maximum number of records to return from the collection. Default value of limit is 50.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: A domain name or comma-delimited list of domain name.
+      isArray: false
+      name: domains
+      required: false
+      secret: false
+    - default: false
+      description: A URL or comma-delimited list of URL.
+      isArray: false
+      name: urls
+      required: false
+      secret: false
+    - default: false
+      description: An IP address.
+      isArray: false
+      name: ip
+      required: false
+      secret: false
+    - default: false
+      description: A SHA-256 hash.
+      isArray: false
+      name: sha256
+      required: false
+      secret: false
+    - default: false
+      description: A threat name or comma-delimited list of threat name.
+      isArray: false
+      name: threats
+      required: false
+      secret: false
+    - default: false
+      description: A threat type or comma-delimited list of threat type.
+      isArray: false
+      name: threat_types
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: An AMP disposition string.
+      isArray: false
+      name: amp_disposition
+      predefined:
+      - clean
+      - malicious
+      - unknown
+      required: false
+      secret: false
+    - default: false
+      description: An identity type or comma-delimited list of identity type.
+      isArray: false
+      name: identity_types
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: A verdict string.
+      isArray: false
+      name: verdict
+      predefined:
+      - allowed
+      - blocked
+      - proxied
+      required: false
+      secret: false
+    - default: false
+      description: The page number. Default Page number is 1.
+      isArray: false
+      name: page
+      required: false
+      secret: false
+    - default: false
+      description: The number of requested results per page.
+      isArray: false
+      name: page_size
+      required: false
+      secret: false
+    deprecated: false
+    description: List of categories ordered by the number of requests made matching the categories in descending order.
+    execution: false
+    name: umbrella-reporting-category-list
+    outputs:
+    - contextPath: UmbrellaReporting.Category.count
+      description: Count
```

Comment:
This description is not clear.  What is this the count of?

Reply:
@ShacharKidor Modified